dali/operators/decoder/image_decoder.cc

```diff
@@ -108,14 +108,14 @@ the DALI pipeline and should be found empirically. More details can be found at
 Applies **only** to the ``mixed`` backend type in NVIDIA Ampere GPU architecture.
 
 The hint is used to preallocate memory for the HW JPEG decoder.)code",
-      0)
+      0u)
```

Comment:
I'm not sure about the unsigned default, how it interacts with Python, and than you access it by `GetArgument<int>` - so signed value. I would stick with signed default + DALI_ENFORCE(... >= 0, "Negative hint not allowed"); or something like that.

Reply:
Done